ionic2/app/components/poke-details/poke-stats.component.ts

```diff
@@ -0,0 +1,15 @@
+import { Component, Input } from '@angular/core';
+import { Pokemon } from "../../models/pokemon";
```

Comment:
Can we use single quotes here? Also for the `@Input("pokemon")` statement below.